data_relay.py

```diff
@@ -8,9 +8,12 @@
 from comm_server import TelemetryFactory, ProducerToManyClient
 from telem_producer import TelemetryProducer
 
-db_type = (
+db_type =(
```

Comment:
```
db_type = {
  HIGH_FREQ: (
     ('d', 'lat'),
     ...
  ),
  LOW_FREQ: (
    (...))
  ...
}
```

Unpack the first two bytes separately, then unpack the remainder using the dereferenced data structure. This means you can have arbitrary clients, not every sender has to know about every type.